### PR TITLE
feat: add support for Kubernetes 1.16.13

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:
See https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#changelog-since-v11612

**Issue Fixed**:

**Requirements**:

- [x] Kubernetes artifacts built and pushed by Azure Pipelines
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by mboersma** [comment]
/hold

This has a temporary commit for testing which needs to be discarded after it passes CI.

File: pkg/api/vlabs/validate_test.go [hunk 1]
```diff
@@ -5272,7 +5272,7 @@ func TestValidateAzureStackSupport(t *testing.T) {
 			cs.Properties.OrchestratorProfile.OrchestratorVersion = "1.16.11"
 			if err := cs.Validate(false); !helpers.EqualError(err, test.expectedErr) {
 				t.Logf("scenario %q", test.name)
-				t.Errorf("expected error: %v, got: %v", test.expectedErr, err)
+				t.Logf("FIXME: expected error: %v, got: %v", test.expectedErr, err)
```
Comment (mboersma) [inline pkg/api/vlabs/validate_test.go:5275]: @jadarsie if I follow our usual process, this leaves Azure Stack without a supported 1.16.x version, so I disabled this test. But let me know if you prefer a different approach--we can be flexible about version support if need be.

**Comment by mboersma** [comment]
This [passed CI](https://github.com/Azure/aks-engine/pull/3602/checks?check_run_id=879503582), so I'll strip off the testing commit and it should be ready for review.

File: pkg/api/vlabs/validate_test.go (hunk 1 above)
Comment (jadarsie) [reply pkg/api/vlabs/validate_test.go:5275]: ASH support for v1.16 here #3613, usual process OK

Comment (jackfrancis) [reply pkg/api/vlabs/validate_test.go:5275]: @jadarsie should I merge this as-is, and then #3613 can rebase and undo the error log changes?

Comment (jadarsie) [reply pkg/api/vlabs/validate_test.go:5275]: @JesusAlvarezTorres FYI

Comment (JesusAlvarezTorres) [reply pkg/api/vlabs/validate_test.go:5275]: @jackfrancis go ahead and merge this in, I'll update #3613 when this is in

**Pull request merged**